dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/instrumentation/TraceParentHeaderFormatter.java

```diff
@@ -0,0 +1,35 @@
+package datadog.trace.bootstrap.instrumentation;
+
+import datadog.trace.api.Ids;
+import io.opentracing.SpanContext;
+import java.math.BigInteger;
+import java.util.Arrays;
+
+public class TraceParentHeaderFormatter {
```

Comment:
No tests for this class?

Reply:
There is a test specifically for the format - in the servlet tests.